cmd/terraform.go

```
@@ -16,42 +20,90 @@ var terraformCmd = &cobra.Command{
+
+		actualCmd.SetUsageTemplate(template)
+		cc.Init(&cc.Config{
+			RootCmd:  actualCmd,
+			Headings: cc.HiCyan + cc.Bold + cc.Underline,
+			Commands: cc.HiGreen + cc.Bold,
+			Example:  cc.Italic,
+			ExecName: cc.Bold,
+			Flags:    cc.Bold,
+		})
+
+		actualCmd.Help()
```

golangci-lint (1.62.2)

Error return value of `actualCmd.Help` is not checked

(errcheck)